### ISSUE TITLE
Page progress tracker not working

### ISSUE THREAD

**Issue by j-norwood-young** [remp2020/remp#31]
https://github.com/remp2020/remp/blob/a4e60a16c9ad14048b7a0c21c66c9c3074dd89c6/Beam/resources/assets/js/remplib.js#L837

This line should be `this.post(this.url + "/track/pageview", params);`

**Comment by rootpd** [comment]
Not necessarily. The `sendTrackedProgress` function is being used within event handlers and `this` is not refering to `remplib.tracker` object there, but to the context of emitted event.

That's why we rather used `remplib.tracker.post` which always points to the right function and doesn't get confused by context.

Is current implementation causing you any issues?

**Comment by j-norwood-young** [comment]
My  current implementation was not posting any events until I changed that.
